src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1465,7 +1466,7 @@ public static <E extends SkriptEvent> SkriptEventInfo<E> registerEvent(String na
 
 		String[] transformedPatterns = new String[patterns.length];
 		for (int i = 0; i < patterns.length; i++)
-			transformedPatterns[i] = "[on] " + SkriptEvent.fixPattern(patterns[i]) + " [with priority (lowest|low|normal|high|highest|monitor)]";
+			transformedPatterns[i] = "[on] " + SkriptEvent.fixPattern(patterns[i]) + EVENT_PRIORITY_SYNTAX;
```

Comment:
Why does this look like it's still sticking the pattern on the end?

Reply:
I'm lost on what that part fully does, but in the html generator file the value gets replaced with an empty string

Reply:
This part is a must otherwise the whole priority syntax addition that TP worked on is lost 😅
I've only replaced it with static field field to be used in HTMLGenerator to be replaced [here](https://github.com/SkriptLang/Skript/pull/5523/files#diff-6762fbc1d43ae0e614d3d4bde26a31d26ddfb076dab557078c1f66f9dc14a80eR666)